packages/components/src/pages/ChatSearch.vue

```diff
@@ -343,34 +344,28 @@ export default {
         };
 
         const onComplete = () => {
-          interp.dispose();
           this.searching = false;
           onProjectContextComplete();
+          systemMessage.complete = true;
           resolve();
         };
 
         const onError = (error) => {
-          interp.dispose();
           onComplete();
           this.$refs.vchat.onError(error, systemMessage);
           reject();
         };
 
-        const interp = new CompletionInterpolation();
```

Comment:
Note the warning about the unused import. So it probably makes senes to drop the file.